### PR TITLE
chore: ignore cassette diffs

### PR DESCRIPTION
**Pull request by Justintime50**
# Description

Ignore cassette diffs locally (line 1) and collapse on GitHub (line 2) for easier review.

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc.)

### CODE REVIEW

**Comment by nwithan8** [comment]
Diff stats for this PR still reflect the changes made to the cassette...

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* text=auto
```
Comment (nwithan8) [inline .gitattributes:1]: See comment above.

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (Justintime50) [reply .gitattributes:1]: You cannot remove the lines added/removed via this method. The "stats" that GitHub's docs talk about is the language breakdown. They built `linguist` to determine the language makeup of a repo, so cassettes won't count towards that. But the bigger thing this solves is that now cassettes will always be collapsed by default and will not show the actual diff on GitHub unless you open up the file.

Comment (nwithan8) [reply .gitattributes:1]: Bummer, so still big green numbers. ¯\_(ツ)_/¯

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**